app/backend/src/clj/teet/backup/backup_ion.clj

```diff
@@ -300,6 +302,14 @@
            :asset-old->new (result "assets.edn"))))
 
 
+(defn- write-backup-to-zip-file [conn file]
+  (with-open [out (io/output-stream file)]
+    (output-zip out
+                ["transactions.edn" (partial output-all-tx conn)]
+                (when (environment/feature-enabled? :asset-db)
+                  ["assets.edn" (partial output-all-tx
+                                         (environment/asset-connection))]))))
+
```

Comment:
This is refactored from `backup*` into a separate function for easy REPL backup.